docs/tutorials.rst

```diff
@@ -6,6 +6,7 @@ Tutorials
 .. toctree::
    :maxdepth: 1
 
+   the-training-cookbook 
```

Comment:
Can we put this in Advanced guides section? I don't think this belongs in the 101 section.

Reply:
I disagree.

Reply:
> but it's way more basic than the stuff in Advanced Guides

I disagree. Given that this uses sharding, mutable arrays, jit, pytrees, etc which are pretty advanced. It should come after we explain all those concepts. This is not the first time users should see when they want to learn JAX (atleast I wouldn't want to see this).

Reply:
Advanced guides it is! Done